src/patternfly/components/MenuToggle/menu-toggle.scss

```
@@ -172,6 +178,14 @@
       border: 0;
     }
   }
+
+  &.pf-m-full-height {
+    --pf-c-menu-toggle--m-expanded--after--BorderBottomWidth: var(--pf-c-menu-toggle--m-full-height--m-expanded--after--BorderBottomWidth);
```

did you mean to set the top border too?

```suggestion
    --pf-c-menu-toggle--before--BorderTopWidth: var(--pf-c-menu-toggle--m-full-height__toggle--before--BorderTopWidth);
    --pf-c-menu-toggle--m-expanded--after--BorderBottomWidth: var(--pf-c-menu-toggle--m-full-height--m-expanded--after--BorderBottomWidth);
```